<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWin</class>
 <widget class="QWidget" name="GameWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="batty">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>500</y>
     <width>60</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imgs.qrc">:/new/images/img/蝙蝠侠.gif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="indent">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="score">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正喵呜体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>560</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正喵呜体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Life:</string>
   </property>
  </widget>
  <widget class="QLabel" name="life1">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>560</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imgs.qrc">:/new/images/img/cat.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="life2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>560</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imgs.qrc">:/new/images/img/cat2.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="life3">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>560</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imgs.qrc">:/new/images/img/cat3.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>score</zorder>
  <zorder>label_3</zorder>
  <zorder>life1</zorder>
  <zorder>life2</zorder>
  <zorder>life3</zorder>
  <zorder>batty</zorder>
 </widget>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
